index.html

```diff
@@ -122,10 +122,16 @@
       illuminate favorable or unfavorable consequences for their needs.
     </p>
     <p>
-      While this rubric allows the evaluation of many aspects of decentralization of
-      a DID Method, it is not exhaustive, and does not cover other factors that may
-      affect selection or adoption of a particular Method, such as privacy, security,
-      reliability, or efficiency.
+      This rubric is a collection of criteria for creating Evaluation
+      Reports that assist you, the Evaluator, in choosing a DID
+      Method.
+
+      While this rubric assists in the evaluation of many aspects of a
+      DID Method, it is not yet exhaustive.
+
+      Criteria are refereed by the rubric editors, with a bias to
+      inclusion when well-formed and not irrelevant, but Evaluation
+      Reports invite opinions from anyone.
```

Comment:
Umm... this framing feels odd. I'm on the fence about mentioning Evaluation Reports as the purpose of the Rubric. In your first use of that, it was technically accurate, but here I mis-read it the first time thinking that you were saying that the Rubric contains evaluation reports and hence, that the Rubric invites opinions from anyone.

We don't.

We invite suggestions for improvements to the set of criteria. Not for people to share their "opinions". That's not what the Rubric is for.

Rather, it is to help people MAKE opinions based on evaluating criteria that practitioners in the industry believe to be useful.

Reply:
In co-editor conversation on this topic, the Rubric itself exists, independent of evaluations, to help people understand which DID Methods may be most appropriate for their needs.

Reply:
```suggestion
```

This is the better thread to reply to.

Reply:
that suggestion failed.